lib/screens/map_view.dart

```diff
@@ -85,18 +88,43 @@ class _MapViewScreenState extends State<MapViewScreen> {
           MarkerLayerOptions(
             markers: [
               Marker(
-                  width: 40.0,
-                  height: 40.0,
+                  width: 110.0,
+                  height: 110.0,
                   point: latlng.LatLng(locationData?.latitude ?? 51.5,
                       locationData?.longitude ?? -0.09),
-                  builder: (ctx) => const Beastie())
+                  builder: (ctx) => const Avatar()),
+              spawnBeastie(),
+              spawnBeastie(),
+              spawnBeastie()
             ],
           ),
         ],
       );
     }
   }
 
+  Marker spawnBeastie() {
+    var random = Random();
+    double longitudeRange = 0.00085;
+    double latitudeRange = 0.00088;
+    int randNum = random.nextInt(2);
+    int sign = 0;
+    if (randNum == 0) {
+      sign = -1;
+    } else {
+      sign = 1;
+    }
+    double randomLon = sign * (random.nextDouble() * longitudeRange + .00015);
```

Comment:
Where do these constants come from? The .00015 and .00032? It would be good to put those in a variable that can remind us why we have those values

Reply:
You're right, I totally forgot I was planning to go back and remove those magic numbers before I made the PR :/ I will fix it